pipeline_dp/dp_engine.py

```diff
@@ -12,16 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """DP aggregations."""
-import collections
 import dataclasses
 import functools
 from typing import Any, Callable, Tuple
-import numpy as np
 
 import pipeline_dp
 from pipeline_dp import combiners
 import pipeline_dp.contribution_bounders as contribution_bounders
 import pipeline_dp.report_generator as report_generator
+import pipeline_dp.utils as utils
```

Comment:
I don't feel great about creating a utils module, maybe we can be more clear about what we are refactoring here. Do we expect to refactor away more statistics/probability-related functions? Then, maybe we can rename it to something like `stats_helpers`/`stats_utils`? If we don't expect any more utils, maybe we can just keep it in DPEngine?

Reply:
Thanks, good point! I don't expect now other functions. But there will be call of this function in contribution_bounder.py in the next PR (WIP).
I've renamed to sampling_utils.py, which is much better represents what this code doing